chroma-manager/tests/integration/shared_storage_configuration/test_iml_diagnostics.py

```diff
@@ -0,0 +1,21 @@
+
+from testconfig import config
+from tests.integration.core.chroma_integration_testcase import ChromaIntegrationTestCase
+
+
+class TestIMLDiagnostics(ChromaIntegrationTestCase):
+    TEST_SERVER = config['lustre_servers'][0]
+
+    def test_diagnostics_with_all_logs(self):
+
+        # Generate the diagnostics from the server
+        run_result = self.remote_operations.run_iml_diagnostics(self.TEST_SERVER, verbose=True)
+        self.assertEqual(run_result.rc, 0,
+                         'IML Diagnostics failed with the following error: %s' % run_result.stderr)
+
+    def test_diagnostics_without_all_logs(self):
+
+        # Generate the diagnostics from the server
+        run_result = self.remote_operations.run_iml_diagnostics(self.TEST_SERVER, verbose=False)
+        self.assertEqual(run_result.rc, 0,
+                         'IML Diagnostics failed with the following error: %s' % run_result.stderr)
```

Comment:
Should we expand these tests, or write new ones to (spot) test the content of the report?

Reply:
Yes, but they should live in the iml-diagnostics repo when we do so (and I didn't realize we had these 😄  )